nohup.out

```diff
@@ -0,0 +1,2 @@
+Starting Storage......SUCCESS
+Starting Grakn Core Server......SUCCESS
```

Comment:
I don't think this is supposed to be commit, right, @vmax ?

Reply:
@haikalpribadi good catch! addressed in https://github.com/graknlabs/docs/pull/344/commits/c73686611e2e1b4396c913ce6bd345cbea23a107